denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/panels/mod_online_inv.dsc

```diff
@@ -0,0 +1,53 @@
+# -- ONLINE PLAYERS PANEL --
+mod_online_inv:
+  type: inventory
+  debug: false
+  title: <&6>A<&e>MP <&f>– <&a><server.online_players.size> online.
+  inventory: CHEST
+  size: 54
+  definitions:
+    border: <item[light_blue_stained_glass_pane].with[display_name=<&r>]>
+    close: <item[red_stained_glass_pane].with[display_name=<&c><&l>Close;nbt=<list[to/close]>]>
+  procedural items:
+    - define inventory:<list>
+    # Loop over list of online players.
+    - foreach <server.online_players> as:player:
+      # Match item display name and lore to information about the online player.
+      - define name:<[player].name>
+      - define skin:<[player].name>
```

Comment:
for `set` data actions using `define`, the syntax is preferred `- define <name> <object>`